posthog/models/person/person.py

```diff
@@ -138,17 +138,15 @@ class PersonOverride(models.Model):
 
     class Meta:
         constraints = [
-            models.UniqueConstraint(fields=["team", "old_person_id"], name="unique override per old_person_id"),
             models.CheckConstraint(
                 check=~Q(old_person_id__exact=F("override_person_id")),
                 name="old_person_id_different_from_override_person_id",
             ),
         ]
 
     id = models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")
-    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
```

Comment:
People are already FKs to a table (`PersonDistinctId`) that has a `team`, is was redundant.